processes/code-styles/swift/swift-style-guide-en.md

```diff
@@ -1,59 +1,269 @@
+    /* ... */
+}
+```
+
+* **2.19** Follow Xcode's recommended indentation style (i.e. your code should not change if CTRL-I is pressed). When declaring a function that spans multiple lines, prefer using that syntax to which Xcode, as of version 7.3, defaults.
+
+```swift
+// Xcode indentation for a function declaration that spans multiple lines
+func myFunctionWithManyParameters(parameterOne: String,
+                                  parameterTwo: String,
+                                  parameterThree: String) {
+    // Xcode indents to here for this kind of statement
+    print("\(parameterOne) \(parameterTwo) \(parameterThree)")
+}
+
+// Xcode indentation for a multi-line `if` statement
+if myFirstValue > (mySecondValue + myThirdValue)
+    && myFourthValue == .someEnumValue {
+
+    // Xcode indents to here for this kind of statement
+    print("Hello, World!")
+}
+```
+
+* **2.20** When calling a function that has many parameters, put each argument on a separate line with a single extra indentation.
+
+```swift
+someFunctionWithManyArguments(
+    firstArgument: "Hello, I am a string",
+    secondArgument: resultFromSomeFunction(),
+    thirdArgument: someOtherLocalProperty)
+```
+
+* **2.21** When dealing with an implicit array or dictionary large enough to warrant splitting it into multiple lines, treat the `[` and `]` as if they were braces in a method, `if` statement, etc. Closures in a method should be treated similarly.
+
+```swift
+someFunctionWithABunchOfArguments(
+    someStringArgument: "hello I am a string",
+    someArrayArgument: [
+        "VIPER is a super-duper architecture",
+        "but MVC is better"
+    ],
+    someDictionaryArgument: [
+        "key 1": "value",
+        "key 2": "another value"
+    ],
+    someClosure: { parameter1 in
+        print(parameter1)
+    })
+```
+
+* **2.22** Prefer using local constants or other mitigation techniques to avoid multi-line predicates where possible.
+
+**Preferred**
+
+```swift
+// PREFERRED
+let firstCondition = x == firstReallyReallyLongPredicateFunction()
+let secondCondition = y == secondReallyReallyLongPredicateFunction()
+let thirdCondition = z == thirdReallyReallyLongPredicateFunction()
+if firstCondition && secondCondition && thirdCondition {
+    // do something
+}
+```
+
+**Not Preferred:**
+
+```swift
+if x == firstReallyReallyLongPredicateFunction()
+    && y == secondReallyReallyLongPredicateFunction()
+    && z == thirdReallyReallyLongPredicateFunction() {
+    // do something
+}
+```
+
+* **2.23** There **SHOULD** be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
+
+* **2.24** Semicolons. Swift does not require a semicolon after each statement in your code. They are only required if you wish to combine multiple statements on a single line.
```

Comment:
**2.24** and **2.25** duplicate each other. Also example for **2.25** is incomplete. There is only one statement, but the paragraph relates to dozens of them.